### DISCUSSION TITLE
Account user name not listed by filter 'author: '

### DISCUSSION THREAD

**Discussion by Ricky-Tigg** [community/community#38855] [Projects and Issues] [answered]
### Select Topic Area

Bug

### Body

**object**: web application served at https://github.com/orgs/community.

Hello. Filter _author:_ does not list my account user name which would allow me to get a view of the tickets of which i am the author. Expected listing: _author:<my_account_user_name>_ 

![github com_community_filter_by_author](https://user-images.githubusercontent.com/26058215/201326104-5d68c4f3-da26-4d47-aede-afa0ca5767e7.png)

**Comment by jfullstackdev** [comment, marked as answer]
you mean this Discussion Community ? you don't need to search, it's already provided

you must be log in and go to this, it will list all of your discussions, whether you are the author or the commenter

![image](https://user-images.githubusercontent.com/47092464/201475332-ad2e8f12-2dcb-4752-9186-757e61daedac.png)

https://github.com/discussions

as you can see on top, the search is already there by default, so you don't need to worry

**Reply by Ricky-Tigg** [reply]
Thank you. Discussion Community –correct. Alternatively that could be achieved by clicking the **account's profile icon** then _Your discussions_.
